common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -52,6 +52,7 @@ const LiveBlockContainer = ({
 		<article
 			id={`block-${id}`}
 			key={id}
+			className="block"
```

Comment:
The comment included in `ArticleBody` was great, can we do something similar here as well?

Reply:
Sure, I've added a comment!